src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -64,6 +64,10 @@
             <MenuItem mnemonicParsing="false" onAction="#menuItemConvertToCdfClicked"
               text="Convert CVRs in Current Config to CDF"/>
           </Menu>
+          <Menu mnemonicParsing="false" text="Help">
+            <MenuItem mnemonicParsing="false" onAction="#menuItemOpenHelpClicked"
+              text="Open Help Manual"/>
```

Comment:
Nit: Hmm, not sure if we should really call it a "Help Manual". It's kind of more of a "Config File Reference". I'll leave it up to you though.

Reply:
It's aspirational :)